src/pages/Citywide-Stats.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <DefaultLayout>
+    <div class="citywide-stats-page">
+      <h1 class="page-title" tabindex="-1">Citywide Stats</h1>
+      <p>
+        These charts provide an overview of energy and emissions data across all
+        large buildings tracked in the city. Explore trends over time, compare
+        building performance, and identify opportunities for sustainability
+        improvements.
+      </p>
+
+      <div
+        class="graphs-container"
+        aria-label="citywide data and info for all buidings we track"
+      >
+        <article
+          v-for="graph in graphConfigs"
+          :key="graph.containerId"
+          :aria-label="`${graph.title} - graph and info`"
+        >
+          <div class="graph-info">
+            <h2>{{ graph.title }} ({{ graph.yAxisLabel }})</h2>
+            <p v-for="(paragraph, i) in graph.description" :key="i">
+              {{ paragraph }}
+            </p>
+          </div>
+
+          <ScatterGraph
+            :data="graph.data"
+            :y-axis-label="graph.yAxisLabel"
+            :stroke-color="graph.strokeColor"
+            :fill-color="graph.fillColor"
+            :container-id="graph.containerId"
+            :show-grid="true"
+            :title="graph.title"
+          />
+        </article>
+      </div>
+      <p>* According to the Department of Energy, 
+        <a href="https://afdc.energy.gov/data/10310" 
+          target="_blank" 
+          aria-label="department of energy average fuel economy data - opens in a new tab"
+        >
+          the average car gets 24.4 MPG
+        </a>
+        , which requires approximately 102.5 gallons of gas to travel 2,500 miles.
```

Comment:
Wildly trivial thing, but there's an extra space appearing after MPG and before the comma:

<img width="338" height="114" alt="Screenshot 2025-08-19 at 8 11 32 PM" src="https://github.com/user-attachments/assets/6aee4880-865f-40a2-b796-82a4de64ce1d" />